frontend/src/views/LandingPage.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+import Explanation from '../components/Explanation';
 
 function LandingPage(){
     return(
         <div>
-            {/* Insert your created components here */}
+            {/* Insert your created components here */
+                <Explanation/>
+            }
```

Comment:
You can remove the comment here as well and the component does not need to be inside of curly braces.